appender.go

```diff
@@ -130,6 +130,15 @@ func TestAppender(t *testing.T, store types.Storager) {
 			Convey("WriteAppend size should be equal to n", func() {
 				So(n, ShouldEqual, size)
 			})
+
+			content, _ = ioutil.ReadAll(io.LimitReader(randbytes.NewRand(), size))
+			r = bytes.NewReader(content)
+			_, _ = ap.WriteAppend(o, r, size)
+			o, _ = store.Stat(path)
```

Comment:
The object or file may not be retrieved  before `CommitAppend`, like `dropbox`. 
Maybe we can modify `When CommitAppend` case: `WriteAppend` twice and check the content and size(content length) after `Read`.

Reply:
sure